### ISSUE TITLE
Support for partial backups: backup only specific keyspaces/tables, etc.

### ISSUE THREAD

**Issue by alexott** [thelastpickle/cassandra-medusa#116]
It would be really useful to support partial backups of the data, like:

- backup only specific keyspaces or tables (white list);
- exclude specific keyspaces or tables (black list).

This would greatly help in some cases, when cluster contains some data that don't have much value (like temporary data), could be replayed from other sources, or backup could be used for offloading of "cold/inactive" data that could be restored later on table-by-table decision.

P.S. it's similar to #64, but this functionality is really requested by customers, and supported in other products, like, OpsCenter (keyspace-level only).

**Comment by xgerman** [comment]
We would like this as well. Anyway we can help?

**Comment by rzvoncek** [comment]
Hi @xgerman ! I'm doing some grooming of the oldest issues we have and this out caught my attention due to the recent activity.

Also, because implementing this is not really complicated. We'd just need to:
* Add CLI (and perhaps config) options to white/black list the tables to back up.
* Skip the tables being backed up [here](https://github.com/thelastpickle/cassandra-medusa/blob/master/medusa/backup_node.py#L284).

This would still crate and clear snapshots of the not-backed-up tables, but that I don't think matters.

We'd also create situations where one could expect all tables to be restored from the backup, but then don't get them. However, I'm not sure how big of an issue that is.

So to sum up, we'd be very grateful if you could contribute this :)
